Detectors/PHOS/simulation/include/PHOSSimulation/Detector.h

```diff
@@ -166,10 +166,8 @@ class Detector : public o2::Base::DetImpl<Detector>
   */
  private:
   // Geometry parameters
-  Bool_t mCreateCPV;       // Should we create module with CPV
   Bool_t mCreateHalfMod;   // Should we create  1/2 filled module
   Bool_t mActiveModule[6]; // list of modules to create
```

Comment:
better make a constant for the number of modules and use it the the array definition and as upper bound in loops etc